### DISCUSSION TITLE
Follow Behaviour for plane

### DISCUSSION THREAD

**Discussion by iamvaibhavrai** [brianzinn/react-babylonjs#272] [General]
Is follow behavior implemented in this library? If yes, how can I use it? I cannot find any examples.
I am trying to open a plane (GUI) in front of the user. Is there any other way to achieve that? Once the UI is open, I have restricted user movement. If I can open the plane in front of the user without the follow behaviour,  it will also work.

Thanks in advance.

**Comment by brianzinn** [comment]
what is the imperative version of the follow behaviour you are after?

have a look at this GUI demo - it may help with describing what you are after:
https://brianzinn.github.io/react-babylonjs/examples/gui/2d-gui
